src/app/components/forms/form/helper/terra-form-field.helper.ts

```diff
@@ -245,26 +246,76 @@ export class TerraFormFieldHelper
     }
 
     /**
-     * Recursively parses the defaultValue of a formField and it's children.
-     * @param field
+     * @description Determines the default value of a single #formField. Also considers children of a #formField if no defaultValue is given.
+     * @param formField
+     * @param skipList - optional parameter that skips the list check and returns the defaultValue of the single entry, not a list.
      */
-    public static parseDefaultValue(field:TerraFormFieldInterface):any
+    public static parseDefaultValue(formField:TerraFormFieldInterface, skipList?:boolean):any
```

Comment:
```suggestion
    public static parseDefaultValue(formField:TerraFormFieldInterface, skipList:boolean = false):any
```

Die Schreibweise ist eigentlich egal an dieser Steller, so ist es aber etwas deutlicher was der Standardwert ist 😄